cpu/esp32/include/periph_cpu.h

```diff
@@ -349,15 +350,18 @@ typedef enum {
  * @brief   I2C configuration structure type
  */
 typedef struct {
+    uint8_t module;         /**< I2C module identifier */
     i2c_speed_t speed;      /**< I2C bus speed */
     gpio_t scl;             /**< GPIO used as SCL pin */
     gpio_t sda;             /**< GPIO used as SDA pin */
+    bool scl_pullup;        /**< Pullup enabled for SCL pin */
+    bool sda_pullup;        /**< Pullup enabled for SDA pin */
```

Comment:
```suggestion
    gpio_t scl;             /**< GPIO used as SCL pin */
    gpio_t sda;             /**< GPIO used as SDA pin */
    uint8_t module;         /**< I2C module identifier */
    i2c_speed_t speed;      /**< I2C bus speed */
    bool scl_pullup;        /**< Pullup enabled for SCL pin */
    bool sda_pullup;        /**< Pullup enabled for SDA pin */
```

struct packing

Reply:
It seems that the reordering does not save any byte. Should I change it anyway?

Reply:
Hm I had hoped this might save a few bytes of ROM, but it's no big deal either way